### PR TITLE
style: some style changes for login, create account, and add activity

### PR DESCRIPTION
**Pull request by kdvuong**
screenshot:
add
create
login

### CODE REVIEW

File: app/src/main/res/layout/activity_create.xml [hunk 1]
```diff
@@ -1,47 +1,46 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@drawable/login_background">
+    android:layout_height="match_parent" >
 
-    <ImageView
-        android:id="@+id/signup_title_image_view"
+    <TextView
+        android:id="@+id/textView"
```
Comment (c25vdw) [inline app/src/main/res/layout/activity_create.xml:9]: need a more specific id here

File: app/src/main/res/layout/activity_create.xml [hunk 2]
```diff
@@ -1,47 +1,46 @@
...
+        android:layout_marginTop="48dp"
         android:layout_marginEnd="24dp"
         android:layout_marginRight="24dp"
         app:counterEnabled="true"
         app:counterMaxLength="20"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/signup_title_image_view">
+        app:layout_constraintTop_toBottomOf="@+id/textView">
```
Comment (c25vdw) [inline app/src/main/res/layout/activity_create.xml:36]: same as above

File: app/src/main/res/layout/activity_create.xml [hunk 3]
```diff
@@ -136,26 +135,42 @@
 
     <Button
         android:id="@+id/signup_create"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
```
Comment (c25vdw) [inline app/src/main/res/layout/activity_create.xml:138]: just wondering though: is this supposed to be 0dp?

File: app/src/main/res/layout/fragment_library.xml [hunk 4]
```diff
@@ -28,11 +28,12 @@
...
-        app:backgroundTint="@color/design_default_color_primary_dark"
+        android:layout_marginEnd="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginBottom="16dp"
+        app:backgroundTint="@color/colorPrimary"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:srcCompat="@android:drawable/ic_input_add" />
+        app:srcCompat="@android:drawable/ic_input_add"
+        app:tint="#FFFFFF" />
```
Comment (c25vdw) [inline app/src/main/res/layout/fragment_library.xml:38]: should we move this to a resource?

File: app/src/main/res/layout/activity_create.xml (hunk 3 above)
Comment (kdvuong) [reply app/src/main/res/layout/activity_create.xml:138]: I think so. I made it so the two buttons are same width, I think android does something automatically.

**Review by c25vdw — APPROVED** [review]
lgtm

**Review by QuocTrungTran — APPROVED** [review]
lgtm!!!

**Pull request merged**